feature/experimental/p4rt/ate_tests/traceroute_packetout_test/packetout_test.go

```diff
@@ -63,6 +64,9 @@ func testPacketOut(ctx context.Context, t *testing.T, args *testArgs) {
 	leader := args.leader
 	desc := "PacketOut from Primary Controller"
 	ttls := []int{0, 1}
+	if *deviations.P4rtTTLHoplimitZero {
```

Comment:
Why do we have a deviation here? As of the latest HLD, we only care about TTL=1. No support for 0 is normal.
Can we please update this PR?

Reply:
Sure we will remove TTL0 from this test